public/assets/svg/brand-mark.tsx

```
@@ -0,0 +1,30 @@
+  height = 192,
+  color = '#0B3B2D',
+  ...props
+}) => (
+  <svg
+    xmlns='http://www.w3.org/2000/svg'
+    width={width}
+    height={height}
+    fill={color}
+    viewBox='0 0 340 192'
+    {...props}
+  >
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)